src/components/Publication/Actions/Collect/index.tsx

```diff
@@ -74,7 +75,13 @@ const Collect: FC<Props> = ({ publication, isFullPublication }) => {
         </span>
       </motion.button>
       <Modal
-        title={isFreeCollect ? 'Free Collect' : getModule(publication?.collectModule?.type).name}
+        title={
+          isFreeCollect
+            ? 'Free Collect'
+            : isUnknownCollect
+            ? 'Unknown Collect'
```

Comment:
**suggestion:** I think we don't need this condition, it will bet automatically populated from `getModule` 🤔